test/unit/middlewares/tasks-validator.test.js

```diff
@@ -333,13 +333,14 @@ describe("getTasks validator", function () {
     expect(nextMiddlewareSpy.callCount).to.be.equal(1);
   });
 
-  it("should pass the request when correct parameters are passed: page, dev, status and size", async function () {
+  it("should pass the request when correct parameters are passed: page, dev, status, sixe and searchterm", async function () {
```

Comment:
can you add a test that it passes w/o the query term as well.

Reply:
done